### PR TITLE
Translated browser-apis.md

### PR DESCRIPTION
**Pull request by Prabhu646**
Addresses #1

### CODE REVIEW

File: docs/docs/browser-apis.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 ---
 title: Gatsby Browser APIs
-description: Documentation about leveraging standard browser APIs within Gatsby
+description: Documentation sur l'utilisation des API de navigateur standard dans Gatsby
```
Comment (mindsers) [inline docs/docs/browser-apis.md]: Is it the browser that is standard? Or the APIs?

```suggestion
description: Documentation sur l'utilisation des APIs standards de navigateur dans Gatsby
```

File: docs/docs/browser-apis.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 jsdoc: ["gatsby/src/utils/api-browser-docs.js"]
 apiCalls: BrowserAPI
 showTopLevelSignatures: true
 ---
 
 ## Usage
 
-Implement any of these APIs by exporting them from a file named `gatsby-browser.js` in the root of your project.
+Implémentez toute de ces API en les exportant à partir d'un fichier nommé `gatsby-browser.js` dans la racine de votre projet.
```
Comment (mindsers) [inline docs/docs/browser-apis.md]:
```suggestion
Implémentez l'une de ces APIs en les exportant à partir d'un fichier nommé `gatsby-browser.js` à la racine de votre projet.
```

**Review by mindsers — CHANGES_REQUESTED** [review]
Thank you for your contribution! It is really appreciated.  ❤️ 🎉 

There are some stuffs we want to talk about before merging your code. 🚀

File: docs/docs/browser-apis.md (hunk 1 above)
Comment (Prabhu646) [reply docs/docs/browser-apis.md]: Right! Making the changes.

**Comment by Prabhu646** [comment]
Made those changes

**Review by dantereve — APPROVED** [review]
Thanks for your help, looks good to me! 

It looks like it's your very first contribution to Gatsby, good job!  🚀If you want to take some other page to translate when you have some time, feel free to do so 😅.

**Comment by Prabhu646** [comment]
@dantereve Thankyou! Will surely do

**Review by mindsers — APPROVED** [review]

**Pull request merged**